### PR TITLE
Custom styles and fixing issues.

### PR DESCRIPTION
**Pull request by bgood**
Added a custom style for the linter and fixed a handful of issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +20,13 @@
...
 ## Table of Contents
 
--   [Table of Contents](#table-of-contents)
--   [Overview](#overview)
--   [Architecture](#architecture)
--   [Deployment](#deployment)
--   [Configuration](#configuration)
--   [Licensing](#licensing)
--   [Contributing](#contributing)
+*   [Table of Contents](#table-of-contents)
```
Comment (davidcueva) [inline README.md:23]: The tables of contents are generated by VSCode
Can we exclude the list symbol change from - to * ?

File: code-of-conduct.md [hunk 2]
```diff
@@ -14,10 +14,10 @@ We value diverse opinions, but we value respectful behavior more.
 
 Respectful behavior includes:
 
-* Being considerate, kind, constructive, and helpful.
-* Not engaging in demeaning, discriminatory, harassing, hateful, sexualized, or
+*   Being considerate, kind, constructive, and helpful.
```
Comment (davidcueva) [inline code-of-conduct.md:17]: These are standard files that are included verbatim.
I think they should be excluded from the linter review.

**Review by davidcueva — CHANGES_REQUESTED** [review]
Hi Ben, I added a couple of comments, please check.

File: README.md (hunk 1 above)
Comment (bgood) [reply README.md:23]: Which plugin are you using to generate the TOC?

Comment (davidcueva) [reply README.md:23]: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one
With SlugifyMode set to GitHub

Comment (bgood) [reply README.md:23]: Would you be ok with changing the `markdown.extension.toc.unorderedList.marker` setting to `*`? That cause TOC changes and updates to be generated using `*`.

Settings list: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one#supported-settings

File: code-of-conduct.md (hunk 2 above)
Comment (bgood) [reply code-of-conduct.md]: Updated the linter action to ignore the Code of Conduct file.

**Review by davidcueva — APPROVED** [review]

**Pull request merged**